test/OrchardCore.Tests/Modules/OrchardCore.OpenId/OpenIdApplicationStepTests.cs

```diff
@@ -22,17 +22,17 @@ public async Task OpenIdApplicationCanBeParsed(string recipeName, OpenIdApplicat
             m.FindByClientIdAsync(
                 It.IsAny<string>(),
                 It.IsAny<CancellationToken>()))
-            .Returns(
-                new ValueTask<object>(actual));
+            .ReturnsAsync(
+                await ValueTask.FromResult(actual));
```

Comment:
These can't be right, `await` without `async`, should I check?

Reply:
@sebastienros please check.

Reply:
I updated the test